packages/console-client/src/client/http/axios/errors.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+class InvalidArgumentError extends Error {
+  constructor(message: string) {
+    super(message)
+    this.name = 'InvalidArgumentError'
+  }
+}
+
+
+const InvalidArgumentErrorThrower = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  AssertNotFalsy: (propName: string, prop: any) => {
```

Comment:
maybe only `Assert` is enough; any new Assert will have its specific suffix (e.g. `AssertEqual` etc...

Reply:
> or are you suggesting that Assert is inherently telling the consumer that not falsy is the default assertion for a parameter to be valid?

this